extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -103,6 +107,19 @@ export class AmpSidebar extends AMP.BaseElement {
     this.registerAction('toggle', this.toggle_.bind(this));
     this.registerAction('open', this.open_.bind(this));
     this.registerAction('close', this.close_.bind(this));
+
+    installGlobalClickListenerForDoc(this.getAmpDoc()).addBeforeHandler(e => {
```

Comment:
If we do something like this, I'd recommend we instead expose `onNavigate(href, target, isLocal)` method or something so that we don't have to look for all the same `A` and its parameters. Alternatively, I don't see an issue to just add a click listener directly on the `sidebar` itself. Why do we need to listen for global clicks anyway?